environment.sh

```diff
@@ -36,52 +38,95 @@ done
 # Parse environment name
 env_name=$(basename "`pwd`")
 env_name+="_$env_type"
+branch_name=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
So this is all predicated on having the desired branch already checked out, correct? Is that the desired behavior? Is the goal that the user would originally have created this env from this very shell script and then never switched branches? I wonder if there needs to somehow be protection against switching branches so they're not inadvertenly running this updater on a bad branch to begin w/